x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -990,8 +1004,17 @@ export function overridePackageInputs(
     errors = [...errors, ...responseFormattedValidationErrors];
   }
 
-  if (dryRun && errors.length) {
-    return { ...resultingPackagePolicy, errors };
+  if (errors.length) {
+    if (dryRun) {
+      return { ...resultingPackagePolicy, errors };
+    }
+
+    throw new Error(
+      i18n.translate('xpack.fleet.packagePolicyInvalidError', {
+        defaultMessage: 'Package policy is invalid: {errors}',
+        values: { errors: errors.map(({ key, message }) => `${key}: ${message}`).join('\n') },
```

Comment:
I wasn't sure how to surface multiple error messages in this case where we throw a validation error, so I'm just joining all the potential validation errors together w/ newlines here. Definitely not ideal. Would love any input here if anyone has thoughts.

Reply:
I don't have a good suggestion on how to throw multiple errors like this (unless you return a status-like object instead of an `Error`), but just FYI we have a class called `IngestManagerError` that we extend and use for different types of server-side errors

Reply:
Not suggesting adding a new error format to kibana but I always liked the VError [MultiError](https://github.com/joyent/node-verror#reference-multierror) format, where there was a top level message which has the first message plus the count of other errors, plus an `errors` key with each individual error object in.